sdk/src/main/scala/com/horizen/account/abi/ABIDecoder.scala

```diff
@@ -0,0 +1,42 @@
+package com.horizen.account.abi
+
+import org.web3j.abi.datatypes.Type
+import org.web3j.abi.{DefaultFunctionReturnDecoder, TypeReference}
+
+import java.util
+import scala.util.Try
+
+object ABIDecoder {
```

Comment:
I ahve a perception that ABI utilities must be a java classes and interfaces, since they can be used for custom message processors on application level.